src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/allelespecific/AS_QualByDepth.java

```diff
@@ -78,7 +76,9 @@ public List<VCFCompoundHeaderLine> getRawDescriptions() {
     public Map<String, Object> annotate(final ReferenceContext ref,
                                         final VariantContext vc,
                                         final AlleleLikelihoods<GATKRead, Allele> likelihoods ) {
-        return Collections.emptyMap();
+        // first vc is used for the annotation and the second vc here is used just to get the alleles, so in this case we can pass the same vc for both
+        Map<String, Object> annotation = finalizeRawData(vc, vc);
```

Comment:
wait so was this just totally broken except in combineGVCFs?

Reply:
I think so, but I'm worried that I'm missing something and this was left unimplemented intentionally for some reason. But the behavior from VariantAnnotator was undesirable in that if you asked for AS_QD it just wouldn't do anything.